crates/miden-testing/src/kernel_tests/tx/test_note.rs

```diff
@@ -672,7 +672,100 @@ fn test_get_note_serial_number() -> anyhow::Result<()> {
 }
 
 #[test]
-fn test_get_inputs_hash() -> anyhow::Result<()> {
+fn test_build_recipient() -> anyhow::Result<()> {
+    let tx_context = TransactionContextBuilder::with_existing_mock_account().build()?;
+
+    // Create test script and serial number
+    let note_script = ScriptBuilder::default().compile_note_script("begin nop end")?;
+    let serial_num = Word::from([200u32, 201u32, 202u32, 203u32]);
+
+    let code = format!(
+        "
+        use.std::sys
+
+        use.miden::note
+
+        begin
+            # put the values that will be hashed into the memory
+            push.1.2.3.4.4000 mem_storew dropw
+            push.5.6.7.8.4004 mem_storew dropw
+            push.9.10.11.12.4008 mem_storew dropw
+            push.13.14.15.16.4012 mem_storew dropw
```

Comment:
It would be great not to hardcode these values in the masm, but pass them as parameters to the `format!` macro (since they are already defined in Rust)